tests/components/enphase_envoy/test_config_flow.py

```diff
@@ -262,13 +262,16 @@ async def test_form_host_already_exists(
     result2 = await hass.config_entries.flow.async_configure(
```

Comment:
Maybe add a step to try the wrong password first, and then try the right one to make sure it works if they get it wrong the first time?

Reply:
Step has been added